### DISCUSSION TITLE
IS HP drain lowered in lazer?

### DISCUSSION THREAD

**Discussion by deok420** [ppy/osu#20894] [Q&A] [answered]
I switched to lazer today, because I don't want to farm pp anymore and want to focus on tech maps more, and when I started playing maps, it felt like I couldn't die. I thought I had No Fail or something, but no I had no mods on at all. Is this a part of lazer's scoring system, or is it some type of bug(that's there for some reason)?

**Comment by Melancholy-Dev** [comment, marked as answer]
osu!lazer has a different hp system than stable, you no longer lose hp continuously but only when you miss

**Reply by bdach** [reply]
The HP system is different, yes.

> you no longer lose hp continuously but only when you miss

Where did you get this from? I don't think that is anywhere close to true.

---

See also: https://github.com/ppy/osu/search?q=drain&type=discussions

**Reply by deok420** [reply]
It is true :) I tested a bit and from what it looks like, it's absolutely true.

**Reply by bdach** [reply]
HP drain still exists. I have no idea how anyone can conclude it doesn't.

**Reply by deok420** [reply]
No, we're not saying it doesn't exist. Hp isn't affected by 100s & 50s.

**Reply by bdach** [reply]
not true either

https://github.com/ppy/osu/blob/4bc26dbb487241e2bbae73751dbe9e93a4e427da/osu.Game/Rulesets/Judgements/Judgement.cs#L107-L111

**Reply by deok420** [reply]
well then idk, but it's the most accurate
